doc: Fixed Matter documentation related to settings size change. (#10855)
nrfconnect/sdk-nrf: doc/nrf/protocols/matter/getting_started/hw_requirements.rst

.. code-block:: console

   west build -b <build_target> -t partition_manager_report

> For example, for the ``nrf7002dk_nrf5340_cpuapp`` build target, the command looks like follows:

Reviewer: ```suggestion
For example, for the ``nrf7002dk_nrf5340_cpuapp`` build target, the command is as follows:
```

> .. code-block:: console
>    west build -b nrf7002dk_nrf5340_cpuapp -t partition_manager_report

Reviewer: ```suggestion
.. code-block:: console

   west build -b nrf7002dk_nrf5340_cpuapp -t partition_manager_report
```
